README.md

```diff
@@ -280,3 +283,31 @@ We use [Mono's Coding Guidelines](http://www.mono-project.com/community/contribu
 
 We use [Bugzilla](https://bugzilla.xamarin.com/enter_bug.cgi?product=Android) to track issues.
 
+# Maintainer FAQ
+
+## How do I rebuild Mono Native Binaries?
+
+The various Mono runtimes -- all *22* of them (!) -- all store object code
```

Comment:
not sure you want to mention an exact number of 22 here as that might get quickly out of date. Maybe use `over *20* of them (!)` instead.